src/test/scala/org/kibanaLoadTest/helpers/Helper.scala

```diff
@@ -293,4 +295,41 @@ object Helper {
     }
     file
   }
+
+  def checkFilesExist(paths: Path*): Unit = {
+    paths.foreach(path => {
+      val file = new File(path.toString)
+      if (!file.exists()) {
+        throw new RuntimeException(s"File does not exist: ${file.getPath}")
```

Comment:
A more appropriate exception would be `FileNotFoundException` but it's a checked exception.